src/view/dialog/DialogCheckboxes.tsx

```diff
@@ -20,31 +18,35 @@ interface Props {
     venterPaSvarDisabled: boolean;
     values: ('ferdigBehandlet' | 'venterPaSvar')[];
     loading: boolean;
+    isNyopprettet: boolean;
 }
 
-const DialogCheckboxes = ({
+export const DialogCheckboxes = ({
```

Comment:
`DialogCheckboxes` har kun vært brukt av `ManagedDialogCheckboxes`, men den inneholder en del logikk som blir irrelevant for ny dialogtråd.

Bør man gjøre slik det er gjort i denne PR-en, altså exporte `DialogCheckboxes` for å gjenbruke den for nye dialogtråder, eller bør man bare lage en checkbox på egenhånd i `NyDialogForm`?

@tu55eladd Hva tenker du?

Reply:
Tenker man ikke trenger å ta med denne logikken, en enkel checkbox i NyDialogForm holder